### PR TITLE
Update INSTALL on Ruby README.md

### PR DESCRIPTION
**Pull request by tbetbetbe**

### CODE REVIEW

File: src/ruby/README.md [hunk 1]
```diff
@@ -4,31 +4,28 @@ gRPC Ruby
...
 Alpha : Ready for early adopters
 
-INSTALLATION PREREQUISITES
---------------------------
+PREREQUISITES
+-------------
 
-This requires Ruby 2.1, as the RPC API surface uses keyword args.
+- Ruby 2.x. The gRPC API uses keyword args.
+- [homebrew][] on Mac OS X, [linuxbrew][] on Linux.  These simplify the installation of the gRPC C core.
```
Comment (jayantkolhe) [inline src/ruby/README.md]: Can you modify link for linuxbrew to point to https://github.com/Homebrew/linuxbrew#installation

Comment (tbetbetbe) [inline src/ruby/README.md]: Done: PTAL

File: src/ruby/README.md [hunk 2]
```diff
@@ -4,31 +4,28 @@ gRPC Ruby
...
+This will download and run the [gRPC install script][] and then install the latest version of gRPC Ruby gem.  It also installs Protocol Buffers compiler (_protoc_) and the gRPC _protoc_ plugin for ruby.
 
-Installing from source
-----------------------
-
+BUILD FROM SOURCE
+---------------------
+- Clone this repository
 - Build the gRPC C core
 E.g, from the root of the gRPC [git repo](https://github.com/google/grpc)
```
Comment (jayantkolhe) [inline src/ruby/README.md]: [Git repository] instead of [git repo]

Comment (tbetbetbe) [inline src/ruby/README.md]: Done, PTAL

**Pull request merged**